e2e/src/step_definitions/media-viewer.ts

```diff
@@ -127,6 +127,32 @@ Then('I expect to see the document should be downloaded', async function () {
   await downloadPage.waitForDownloadToComplete(file);
 });
 
+When('the user selects the multimedia option', async () => {
+  await multimediaPage.clickMultimedia();
+  await genericMethods.sleep(5000);
```

Comment:
do we need the sleep method? adding 5secs to every click would really slow down the tests
I thought the clickElement method already has waiting time in it